controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -46,7 +46,7 @@ http {
 
     {{ if $cfg.EnableVtsStatus }}
     vhost_traffic_status_zone shared:vhost_traffic_status:{{ $cfg.VtsStatusZoneSize }};
-    vhost_traffic_status_filter_by_set_key $geoip_country_code country::*;
+    vhost_traffic_status_filter_by_set_key {{ if not $cfg.VtsDefaultFilterKey }}$geoip_country_code country::*{{ else }} {{ $cfg.VtsDefaultFilterKey }}{{ end }};
```

Comment:
Please set the default VtsDefaultFilterKey in go

Reply:
Done